doc/subsystems/usb/usb.rst

```diff
@@ -1,5 +1,13 @@
+USB device stack
+################
+
+The USB device stack is split into three layers:
+   * USB Device Controller drivers (hardware dependent)
```

Comment:
Nitpick: lowercase "controller" to make it consistent with lower lines

Reply:
``device`` too :)